plugins/GeoIp2/LocationProvider/GeoIp2/Php.php

```diff
@@ -233,12 +298,18 @@ public function getSupportedLocationInfo()
             switch ($reader->metadata()->databaseType) {
```

Comment:
We should move setting country and continent availability (few lines above) into this if statement. Otherwise it will report `country` as available if even no geoip file is available (or only an isp file is available).